lib/client.js

```diff
@@ -140,6 +140,20 @@ internals.Client.prototype.drop = function (key, callback) {
     this.connection.drop(key, callback);           // Always drop, regardless of caching rules
 };
 
+internals.Client.prototype.dropSegment = function (segment, callback) {
```

Comment:
In my opinion, there needs be some very basic guidance on what is understood under a "segment." Maybe in the _README.md_ file?